base_search_custom_field_filter/models/base.py

```diff
@@ -49,15 +49,22 @@ def get_view(self, view_id=None, view_type="form", **options):
     def get_views(self, views, options=None):
         """Inject fake field definition for having custom filters available."""
         res = super().get_views(views, options)
+
```

Comment:
Please don't insert empty lines inside the method (the same for all the occurrences).